src/components/lib/jitutils/jitutils.h

```diff
@@ -0,0 +1,77 @@
+#include "cos_types.h"
```

Comment:
This lib is just jiting callgates. Jitutil is not a good name. Maybe CallgateJIT or something like that.